modules/base/rendering/renderabletrail.cpp

```diff
@@ -291,6 +295,17 @@ bool RenderableTrail::isReady() const {
     return _programObject != nullptr;
 }
 
+glm::dvec3 RenderableTrail::translationPosition(Time time) const {
+    // Use an empty modelTransform to get the position in local coordinates. Also, the
+    // previous frame time does not matter so we just use time 0
+    const UpdateData data = { {}, time, Time(0.0) };
+    // @TODO (2025-06-09, emmbr) Do we need to also call the update for the translation?
+    // As of now, this borks up the performance of the trail completely, and I'm not sure
+    // the update is needed since we are not using the cached version of the position.
+    //_translation->update(data);
+    return _translation->position(data);
+}
```

Comment:
@alexanderbock @Arohdin I ended up not calling the update function here, since it's no longer needed when the initialization is being done correctly. 

Would be good with a second pair of eyes on this, but calling the update here completely borks up the performance for the trails. And, if I understand the update structure and caching of the transform positions correctly, we shouldn't actually need to call the update as long as we are getting the position from the non-cached version of the position function (which is the one used here, that takes the `UpdateData` struct). Do you agree with this?

Reply:
👍 Is the current comment ok or should it be rephrased? Feel free to come with a suggestion

Reply:
Yea, I think the comment is great. Just didn't want my message to read as if the comment should disappear